src/stylesheets/core/_functions.scss

```diff
@@ -387,10 +387,13 @@ the desired final units (currently rem)
 */
 
 @function spacing-multiple($unit) {
-  $grid-to-rem: ($system-spacing-grid-base * $unit) / $root-font-size-equiv *
-    1rem;
-
-  @return $grid-to-rem;
+  @return 1rem * divide(
+    divide(
```

Comment:
Was there a reason for the extra division instead of something like:

```scss
  @return 1rem * divide(
    $system-spacing-grid-base * $unit,
    $root-font-size-equiv
  );
```

Reply:
Good point! I can't recall exactly why I'd implemented it this way, though I remember having to do some trickery to convert units correctly. Testing the flattened version seems to produce the same output, so I'll simplify it with your suggestion.

Reply:
Simplified in 7f7ee9857.